sampleFiles/clinical/specimen.tsv

```diff
@@ -1,3 +1,2 @@
 program_id	submitter_donor_id	submitter_specimen_id	specimen_acquisition_interval	specimen_anatomic_location	central_pathology_confirmed	tumour_histological_type	tumour_grading_system	tumour_grading_system	tumour_grade	pathological_tumour_staging_system	pathological_stage_group	percent_tumour_cells	percent_proliferating_cells	percent_inflammatory_tissue	percent_stromal_cells	percent_necrosis
-TEST-CA	ICGC_0001.1	sub-sp-pacaau-123	200	Other	No	M-1111/22	Default	Default	aStringValue	Murphy	aStringValue	0.35	0.5	0.6	0.65	0.65
```

Comment:
was this deleted for testing?

Reply:
There were two of the same row, so I deleted one...